srtcore/core.cpp

```diff
@@ -3873,6 +3886,8 @@ bool srt::CUDT::processAsyncConnectRequest(EReadStatus         rst,
             LOGC(cnlog.Warn,
                  log << CONID()
                      << "processAsyncConnectRequest: REJECT reported from processRendezvous, not processing further.");
+
+            sendRendezvousRejection(serv_addr, (request));
```

Comment:
Rendezvous connection, non-blocking mode.
Current master:
- HS Rejection - not sent,
- SHUTDOWN - sent.

This PR:
- HS Rejection - sent, 
- SHUTDOWN - sent.